x-pack/plugins/security_solution/public/flyout/entity_details/host_right/content.tsx

```diff
@@ -44,7 +46,12 @@ export const HostPanelContent = ({
   isPreviewMode,
 }: HostPanelContentProps) => {
   const observedFields = useObservedHostFields(observedHost);
-
+  const getSetupStatus = useCspSetupStatusApi({
+    refetchInterval: 10000,
+  });
+  const notEmptyIndices = useMemo(() => {
```

Comment:
check out the last PR from @JordanSh , he introduced the field in the status API `hasMisconfigurationsFindings` for this usecase. The current logic doesn't cover the 3rd party indexes
https://github.com/elastic/kibana/pull/190285/files#diff-7e52c60ac5b5f65b46ea55ac2d44c1f2c6783499b6057e7ea4cefc32eb9f16dbR405